### ISSUE TITLE
cannot produce the results in table 4

### ISSUE THREAD

**Issue by seyeeet** [baidut/paq2piq#9]
Hello
Thank you for the great work.
I am trying to re-produce the results in table 4 of the paper however I cannot. 
![image](https://user-images.githubusercontent.com/62394093/105606923-f3e8c680-5d69-11eb-9b46-1b320beabd7b.png)

- should we test the model only on the validation set of CLIVE and KonIQ or on the whole dataset?

- can you share the trained models,
P2P-BM (Sec. 4.1) , P2P-RM (Sec. 4.2), and P2P-FM (Sec. 4.4), that can produce those results?

**The results that I get from running the pre-trained model are far less than what is reported on the paper.**

**Comment by baidut** [comment]
Hi, thank you for reporting. 

We faced legal issues trying to make public a considerable number of images. Therefore, we removed those images.
We have to re-split the database and retrain the model after. We didn't share the original pretrained models because the split is different. 

However, the model should give a similar performance. I am writing a journal paper and more pre-trained models will be released soon. Also note the performance also vary with the split of CLIVE and KonIQ, we will report the performance on the whole database instead in the journal paper. 

Thank you again and more discussion is welcome!

**Comment by seyeeet** [comment]
Thanks for the reply. 
I understand that the new version will come out soon, but for the time being, can you please clarify if the provided results in the paper are on the splits of CLIVE and KonIQ or on the whole datasets?

**Comment by baidut** [comment]
Yes, I am writing a document for that. Just uploaded the original pretrained model and will publish the google colab soon

**Comment by baidut** [comment]
@seyeeet  Please follow [this google colab](https://colab.research.google.com/drive/1M2sW86mCBMRkYFa80tI4M85cP8Nrw_kI?usp=sharing) to reproduce the results. Thanks!
